lib/cronitor.rb

```diff
@@ -34,7 +34,7 @@ def initialize(token: nil, opts: {}, code: nil)
   def create
     response = Unirest.post(
       "#{API_URL}/monitors",
-      headers: default_headers.merge 'Content-Type' => 'application/json',
+      headers: default_headers.merge('Content-Type': 'application/json'),
```

Comment:
The additional brackets seem to be important, as some versions of ruby were refusing to compile the code without them. At least on my machine. The string to symbol change was unintentional.